content/sensu-go/5.15/sensuctl/reference.md

```diff
@@ -798,14 +802,71 @@ fetching bonsai asset: sensu/sensu-slack-handler:1.0.3
 added asset: sensu/sensu-slack-handler:1.0.3
 {{< /highlight >}}
 
-To check your Sensu backend for assets which have newer versions available on Bonsai, use `sensuctl asset outdated`. This will print a list of assets installed in the backend whose version is older than the newest version available on Bonsai.
+### Check your Sensu backend for outdated assets
+
+To check your Sensu backend for assets that have newer versions available on Bonsai, use `sensuctl asset outdated`. This will print a list of assets installed in the backend whose version is older than the newest version available on Bonsai.
 {{< highlight shell >}}
 sensuctl asset outdated
           Asset Name                  Bonsai Asset          Current Version  Latest Version
 ----------------------------  ----------------------------  ---------------  --------------
 sensu/sensu-influxdb-handler  sensu/sensu-influxdb-handler       3.1.1            3.1.2
 {{< /highlight >}}
 
+### Extend sensuctl with commands
+
+Use `sensuctl command` to install, execute, list, and delete commands from Bonsai or a URL.
+
+#### Install commands
+
+To install a sensuctl command from Bonsai or a URL, use:
+{{< highlight shell >}}
+sensuctl command install [ALIAS] ([NAME]:[VERSION] | --url [ARCHIVE_URL] --checksum [ARCHIVE_CHECKSUM]) [flags]
+{{< /highlight >}}
+
+Replace `[ALIAS]` with a unique name for the command. `[ALIAS]` is required.
+
+You must specify **either** `[NAME]` (with optional `[:VERSION]`) **or** `--url ARCHIVE_URL --checksum ARCHIVE_CHECKSUM`.
+
+Replace `[NAME]` with the name of the asset from Bonsai.
+`[:VERSION]` is only required if you require a specific version or are pinning to a specific version. If you do not specify a version, sensuctl will fetch the latest version from Bonsai.
+
+Replace `[flags]` with the resource-specific flags you want to use.
+
+For example, to install a command from the [Sensu Go Email Handler Plugin][35] with the [`-t` and -`f` flags][36] to retrieve the "To" and "From" email addresses:
+
+{{< highlight shell >}}
+sensuctl command install [ALIAS] (sensu/sensu-email-handler:0.2.0) -tf
+...
+**NEEDED**
+...
+{{< /highlight >}}
+
+To install a command from a URL:
+
+{{< highlight shell >}}
+sensuctl command install [ALIAS] (--url https://gist.githubusercontent.com/amdprophet/43790e007e949aed19131b100e1576af/raw/0c3a1af85481db2bcfbf9f7be513fa7fc0d0fc18/asset.yml --checksum **NEEDED*)
```

Comment:
When installing a sensuctl command asset via a URL, the URL must point to an asset tarball (e.g. `https://path/to/asset.tar.gz`).

Asset definitions with multiple asset builds are currently only supported via Bonsai.

Reply:
A working example:

```
sensuctl command install command-test --url https://github.com/amdprophet/command-test/releases/download/v0.0.4/command-test_0.0.4_darwin_amd64.tar.gz --checksum 8b15a170e091dab42256fe64ca7c4a050ed49a9dbfd6c8129c95506a8a9a91f2762ac1a6d24f4fc545430613fd45abc91d3e5d3605fcfffb270dcf01996caa7f
```

Reply:
Awesome, thank you!